docs/developers/get_started/getting_started_windows_cmake.md

```diff
@@ -146,11 +146,11 @@ Check out the contents of the 'tools' build directory:
 ```
 
 Translate a
-[MLIR file](https://github.com/google/iree/blob/main/iree/tools/test/iree-run-mlir.mlir)
+[MLIR file](https://github.com/google/iree/blob/main/iree/tools/test/iree-run-module.mlir)
 and execute a function in the compiled module:
 
 ```powershell
-> ..\iree-build\iree\tools\iree-run-mlir.exe .\iree\tools\test\iree-run-mlir.mlir -function-input="i32=-2" -iree-hal-target-backends=vmvx -print-mlir
+> ..\iree-build\iree\tools\iree-run-mlir.exe .\iree\tools\test\iree-run-module.mlir -function-input="f32=-2" -iree-hal-target-backends=vmvx -print-mlir
```

Comment:
The example using `iree-run-mlir.exe` should probably keep using `iree-run-mlir.mlir`.

Though, taking a step back, I'd rather add back a `simple.mlir` or `abs.mlir` at https://github.com/google/iree/tree/main/iree/samples/models that doesn't have the tool-specific naming for use from samples and docs. We can keep the `iree-run-mlir.mlir` and similarly named files at `iree/tools/test/` since those files are testing the tools that match their names.

It would probably be easier for me to just send a PR with the changes I'd like to see there. I'll base my work off of what you have here already.

Reply:
Sent https://github.com/hcindyl/iree/pull/1 with that + my other comments addressed.

Reply:
Thanks! There are a few missing updates in the PR. Keeping the doc up-to-date without tests is not easy 😮‍💨